### ISSUE TITLE
Update on Deprecated API Usage UIWebView 

### ISSUE THREAD

**Issue by aliakhtar-hu** [sendbird/sendbird-ios-framework#65]
As per apple "ITMS-90809: Deprecated API Usage - App updates that use UIWebView will no longer be accepted as of December 2020. Instead, use WKWebView for improved security and reliability."

When running  `grep -r "UIWebView" .`

We are getting  matches that library used UIWebView 
`Binary file ./Pods/SendBirdSDK/SendBirdSDK.framework.dSYM/Contents/Resources/DWARF/SendBirdSDK matches`

In which release we are expecting this to fix? please update

Thanks

**Comment by intoxicated** [comment]
@aliakhtar-hu will look into it

**Comment by intoxicated** [comment]
@aliakhtar-hu 
I searched and confirmed that we do not have any UIWebView usage in our framework. 
I found that your result may result from .dSYM file which don't contain any functional code. This is debugging metadata that’s been stripped from the sendbird framework. The reason why you've found this mention of UIWebView in our dSYM is because we link to UIKit and the version of iOS that may include UIWebViews.
